docker/release/nanoserver/Dockerfile

```diff
@@ -46,3 +46,5 @@ RUN Start-Transcript -path C:\Dockerfile.log -append -IncludeInvocationHeader ;
 
 # Persist %PSCORE% ENV variable for user convenience
 ENV PSCORE='"C:\Program Files\PowerShell\Core\PowerShell.exe"'
+
+ENTRYPOINT ["C:\\Program Files\\PowerShell\\Core\\PowerShell.exe"]
```

Comment:
Typically, powershell.exe is expected to be installed to a versioned folder (`c:\program files\powershell\6.0.0-alpha.17\powershell.exe`), how does `Core` fit into this?